src/components/PuzzleGameContainer/index.jsx

```diff
@@ -0,0 +1,41 @@
+import React, { useState, useEffect } from "react";
+import { Col, Row, Card } from "antd";
+import habitat from "./../../images/puzzleGame/habitat.jpg";
+import yellowSpot from "./../../images/yellow-spot.svg";
+import "./index.css";
+
+// This component contains the list of images the user can choose to solve
+// it also renders the PuzzleGrid component
+// When the user clicks on an image it will be passed to the PuzzleGrid component as a prop
+// PuzzleGrid then renders the puzzle tiles of that image
+
+const PuzzleGameContainer = () => {
+  // currentImage state will be passed to the PuzzleGrid Component
+  const [currentImage, setCurrentImage] = useState(habitat);
+  const images = [habitat, yellowSpot];
```

Comment:
iam not sure but i am felling you should name the images (habitat,yellowSpot) in a different  way, like habitatImg or something ,  when i first read it it looks random then when i saw the import i understand they are an images.

Reply:
+1 suffix img imports with img